.github/workflows/release_gh_pages.yml

```diff
@@ -0,0 +1,114 @@
+name: Release Gh-Pages
+
+on: 
+  push: 
+    tags: '1.*'
```

Comment:
We should likely be careful with this tag: if we bump the version, we would not release the gh pages anymore?

Reply:
Yes, that is a problem
However, we could not find another way to handle releases, a release event needs to be explicitly triggered, as we do farther below, and in our case a release is simply a tagged push to main.

For testing purposes we used tags of the form v1.0... and so on, which allowed us to use "v*" wildcards. We could change it to '*', but this felt more representative of a tag.